src/pydargs/__init__.py

```diff
@@ -166,18 +167,7 @@ def _create_parser(tp: Type[Dataclass], **kwargs: Any) -> ArgumentParser:
     return parser
 
 
-Fct = TypeVar("Fct")
-
-
-def rename(name: str):
-    def _rename(f: Fct) -> Fct:
-        setattr(f, "__name__", name)
-        return f
-
-    return _rename
-
-
-@rename("bool")
+@rename(name="bool")
```

Comment:
Would it be an idea to move the _parse_<...> functions below into `utils.py`, or maybe a `parsers.py`?

Reply:
I think that would be nice, but I would prefer that to be a separate PR.